microsoft-edge/index.yml

```diff
@@ -193,7 +193,7 @@ additionalContent:
             - text: Open developer account
               url: /microsoft-edge/extensions-chromium/publish/create-dev-account
```

Comment:
performance analysis link is broken on landing and hub page though .md isn't included